src/main/java/ch/njol/skript/entity/EntityData.java

```diff
@@ -427,13 +429,25 @@ else if(e instanceof Zombie)
 					((Zombie) e).setBaby(true);
 				else if(e instanceof PigZombie)
 					((PigZombie) e).setBaby(true);
-			}else if(baby.isFalse()){
+				else if (Skript.isRunningMinecraft(1, 16)) {
+					if (e instanceof Piglin)
+						((Piglin) e).setBaby(true);
+					else if (e instanceof Zoglin)
+						((Zoglin) e).setBaby(true);
+				}
+			} else if(baby.isFalse()){
```

Comment:
missing space before the ending brace

Reply:
Will fix this, I dont think it was my line of code, I copy/pasted from above. But I will fix it :)